### ISSUE TITLE
Qt spacing/margin on macOS is larger than in Windows and Linux

### ISSUE THREAD

**Issue by micahflee** [micahflee/onionshare#630]
In macOS, there is much more spacing between widgets in layouts, in the status bar, between the items in the file list, etc., than there is in Windows and Linux. I'm not sure why, but it doesn't look as good and causes UI issues like #625 and #626. I'd like to figure out a way to, across the board, decrease spacing/margins in macOS.

Here's what OnionShare looks like with when sharing 4 files in macOS:

<img width="542" alt="screen shot 2018-02-24 at 11 53 49 am" src="https://user-images.githubusercontent.com/156128/36634508-71ea8b10-195a-11e8-8e5d-9eedcbab2eff.png">

And here's what it looks like in Linux:

![screenshot_2018-02-24_12-00-47](https://user-images.githubusercontent.com/156128/36634518-95cd59ea-195a-11e8-9c9b-70b8836e56a4.png)

**Comment by mig5** [comment]
One way I think we can 'fix' this is to force the GTK style on macOS so it's at least consistent (that would save us our sanity as developers if nothing else!)

In OnionshareGui.__init__:

```
        self.setStyle(QtWidgets.QStyleFactory.create('Fusion'))
```

<img width="542" alt="screen shot 2018-02-25 at 7 40 41 am" src="https://user-images.githubusercontent.com/99173/36634865-7217aa6e-19ff-11e8-8fd0-b9e39c305e2d.png">

You get our awkward scrollbar in the file widget that deals with file widget appearance bugs, but it's *consistent* if nothing else, and the spacing issues are fixed. It also fixes #626 (though I see you pushed a separate fix for that just now in #631)

**Comment by micahflee** [comment]
Oh interesting!

I bet when you're in GTK style mode you click "Add", it will open a GTK style file browser which isn't nearly as nice as the macOS style -- however I'm sure that could be fixed too. I think that "solution" could work, but I'd want to try to use as much macOS style as possible, like for the file browser, and probably for the settings dialog and other dialogs, etc.
